pkg/apis/apps/v1/defaults_test.go

```diff
@@ -173,7 +176,10 @@ func TestSetDefaultDaemonSetSpec(t *testing.T) {
 }
 
 func TestSetDefaultStatefulSet(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.MaxUnavailableStatefulSet, true)()
```

Comment:
There should be a test were the feature is disabled. The field would have to remain `nil`.

Reply:
This still needs to be addressed.